draft-rebs-dnsop-svcb-dane.md

```diff
@@ -128,69 +134,74 @@ The TLSA QNAME is `_443._tcp.api.example.com`.
 
 ## HTTPS AliasMode
 
-Given service https://api.example.com and records:
+Given service URI `https://api.example.com` and records:
 
 ~~~
 api.example.com.     HTTPS 0 svc4.example.net.
-svc4.example.net.    HTTPS 0 xyz.example-cdn.com.
+svc4.example.net.    HTTPS 1 xyz.example-cdn.com.
```

Comment:
I liked this better the other way.  HTTPS does allow the chain to terminate with an AliasMode record.  This way seems too similar to the previous example.

Reply:
Reverted